app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/ColorCorrect.scala

```diff
@@ -149,14 +153,25 @@ object ColorCorrect {
     val maybeEqualize =
       if (params.equalize) Some(HistogramEqualization(_: MultibandTile, rgbHist)) else None
 
+    val rgbBand = (a:Option[Int], b:Option[Int], c: Int) => a.fold(b)(Some(_)).fold(Some(c))(x => Some(x))
```

Comment:
This is just for getting:

- band-specific min/max, if present
- all-bands min/max, if present
- the min/max on the tile otherwise

right?